src/ec/starlabs/it8987/Makefile.inc

```diff
@@ -3,7 +3,7 @@
 PHONY+=add_ite_fw
 INTERMEDIATE+=add_ite_fw
 
-ifeq ($(CONFIG_EC_STARLABS_IT8987),y)
+ifeq ($(CONFIG_EC_STARLABS_IT8987),y) ($(CONFIG_BOARD_STARLABS_LABTOP_CML),y)
```

Comment:
The EC code shouldn't include the `CONFIG_BOARD...` guard.

Both platforms will need to include the EC code and the ASL. It will only be CML board that needs to include the EC firmware.  The `mainboard/starlabs/labtop/Kconfig` file should be amended to have EC_STARLABS_IT8987 and EC_STARLABS_IT8987_BIN and only set the _BIN option when needed.

Reply:
```
ifeq ($(CONFIG_EC_STARLABS_IT8987),y)
all-y += ec.c
smm-$(CONFIG_DEBUG_SMI) += ec.c
endif

ifeq ($(CONFIG_EC_STARLABS_IT8987_BIN), y)

add_ite_fw: $(obj)/coreboot.pre
	$(CBFSTOOL) $(obj)/coreboot.pre write -r EC -f $(CONFIG_EC_STARLABS_IT8987_FW_PATH) -u

else
files_added:: warn_no_ite_fw
endif
```

```
        select EC_STARLABS_IT8987_BIN if BOARD_STARLABS_LABTOP_CML
```

Would `select` make EC_STARLABS_IT8987_BIN=y, or will that need to be added to the config file?